test/test_tensor_creation_ops.py

```diff
@@ -2536,6 +2536,13 @@ def test_arange_inference(self, device):
                                    torch.tensor(3),
                                    torch.tensor(1, dtype=torch.int16)).dtype)
 
+    def test_arange_float_inconsistent_bounds(self):
```

Comment:
Please just add the case in question to https://github.com/pytorch/pytorch/blob/365ac49840105918c604a6b1c7e81c1ca59e37fb/torch/testing/_internal/common_methods_invocations.py#L778-L782